druid/src/widget/image.rs

```diff
@@ -69,7 +69,7 @@ use tracing::{instrument, trace};
 ///
 /// [scaling a bitmap image]: ../struct.Scale.html#pixels-and-display-points
 /// [SVG files]: https://en.wikipedia.org/wiki/Scalable_Vector_Graphics
-/// [`ImageBuf`]: ../druid_shell/struct.ImageBuf.html
+/// [`ImageBuf`]: ../piet/struct.ImageBuf.html
 /// [`FillStrat`]: ../widget/enum.FillStrat.html
 /// [`InterpolationMode`]: ../piet/enum.InterpolationMode.html
```

Comment:
```suggestion
/// [`ImageBuf`]: piet::ImageBuf
/// [`FillStrat`]: crate::widget::FillStrat
/// [`InterpolationMode`]: piet::InterpolationMode
```

Reply:
we can do even better: if these types are in scope, we don't need to include explicit links to them at all, rustdoc will automatically resolve links with the format ```[`Type`]``` if `Type` is in scope.

Reply:
I've updated with those three lines completely removed. All three links do work in the built documentation.